changes/16496.md

```diff
@@ -0,0 +1,11 @@
+# PR #16496 Decouple Mina_block.{Stable.Latest.t, t}
```

Comment:
Not a user-facing change; remove the changelog.

Reply:
Alternatively: describe the memory improvements that we will see when we're writing proofs from blocks to disk. `Stable.Latest.t` is not meaningful to our users.